packages/playground/src/components/ssh_keys/SshDataDialog.vue

```diff
@@ -95,16 +99,39 @@ export default defineComponent({
   setup(props, ctx) {
     const currentKey = ref<SSHKeyData>(props.selectedKey);
     const loading = ref<boolean>(false);
-
+    const hasChanges = ref<boolean>(true);
```

Comment:
As per `showToolTip` logic, I think it should be initialized with false, and you can reset it when dialog is open